app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt

```diff
@@ -273,6 +275,13 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
                 val index = querylanguage.findIndexOfValue(PrefManager.getString(Constant.LANGUAGE, Constant.DEFAULT))
                 querylanguage.setValueIndex(index)
                 querylanguage.summary = querylanguage.entries[index]
+                if (index == 1) setLocalLanguage("de")
```

Comment:
PrefManager.getString(Constant.LANGUAGE, Constant.DEFAULT)

When you have this, why searching from index?

Reply:
I have changed it. Please check it now